params/config.go

```diff
@@ -411,7 +368,12 @@ func (c *ChainConfig) checkCompatible(newcfg *ChainConfig, headHeight *big.Int,
 
 	// Check that the configuration of the optional stateful precompiles is compatible.
 	if isForkIncompatible(c.TxAllowListConfig.Timestamp(), newcfg.TxAllowListConfig.Timestamp(), headTimestamp) {
-		return newCompatError("AllowList fork block timestamp", c.TxAllowListConfig.Timestamp(), newcfg.TxAllowListConfig.Timestamp())
+		return newCompatError("TxAllowList fork block timestamp", c.TxAllowListConfig.Timestamp(), newcfg.TxAllowListConfig.Timestamp())
+	}
+
+	// Check that the configuration of the optional stateful precompiles is compatible.
```

Comment:
Looks like we use the exact same comment for each of the stateful precompiles, can we make each of these comments specific to the relevant precompile?